pisa/utils/hypersurface/hypersurface.py

```diff
@@ -27,9 +27,15 @@
 
 
 import os
-import collections
 import copy
 
+# Handle change over time in `collections` module
+from collections import OrderedDict
+try :
+    from collections.abc import Mapping, Sequence # Required as of py3.10
+except Exception as e :
```

Comment:
Why is there a need for this exception handling?

`Mapping` and `Sequence` are part of `collections.abc` in version [3.8](https://docs.python.org/3.8/library/collections.abc.html) - the lowest python version PISA currently supports. In fact, the `abc` module was introduced in version 3.3.

Furthermore, there are many other PISA modules which make use of the upper (`from collections.abc import Mapping, Sequence`) statement without exception handling.

Unless there's some particular motivation for it in this file, I'd suggest removing the try/except clause here.

Reply:
Sounds good, have removed the older version